xml/ha_storage_protection.xml

```diff
@@ -763,34 +760,27 @@ Timeout (msgwait)  : 40
     <para>
      Usually you should use the above resource with the
      <literal>Filesystem</literal> resource, for example, MD-RAID, LVM, and
```

Comment:
To remove "MDRAID, LVM, and" is more technical appropriate grammar to correlate to "Filesystem" in the sentence ;)

Reply:
You are right :) Done.